src/Microsoft.ML.CpuMath/AlignedArray.cs

```diff
@@ -110,21 +110,23 @@ private void MoveData(int newBase)
             }
         }
 
-        public void CopyTo(Float[] dst, int index, int count)
+        public void CopyTo(Span<Float> dst, int index, int count)
         {
             Contracts.Assert(0 <= count && count <= _size);
             Contracts.Assert(dst != null);
             Contracts.Assert(0 <= index && index <= dst.Length - count);
-            Array.Copy(Items, _base, dst, index, count);
+            for (int i = 0; i < count; i++)
```

Comment:
(I think this code is going away with #1657)

You don't need to copy element by element. Instead you can say `Items.AsSpan(_base + i, count).CopyTo(dst);`